Resources/config/services.yml

```diff
@@ -14,16 +17,18 @@ services:
             - '@markup_job_queue.scheduled'
     jobby:
         alias: markup_job_queue.manager
-    markup_admin_job_queue_recurring_console_command_reader:
+    markup_job_queue.reader.recurring_console_command:
```

Comment:
I think calling it `markup_job_queue.recurring_console_command_reader` would be far clearer.